site/content/docs/5.1/examples/navbars/index.html

```diff
@@ -5,412 +5,1393 @@
+{{< /orange-supra >}}
+
+<div class="b-example-divider"></div>
+
+{{< orange-supra mode="languages" >}}
+{{< /orange-supra >}}
+
+<div class="b-example-divider"></div>
+
+{{< orange-supra mode="actions" >}}
+{{< /orange-supra >}}
+
+<div class="b-example-divider"></div>
+
+<nav class="navbar global navbar-dark bg-dark navbar-expand-md">
+  <div class="container-lg">
+    <a class="navbar-brand" href="#">
+      <img src="/docs/5.1/assets/brand/orange-logo.svg" width="50" height="50" role="img" alt="Boosted" loading="lazy">
+    </a>
+    <div class="d-flex">
+      <ul class="navbar-nav d-md-none flex-row ml-auto">
+        <li class="nav-item">
+          <a href="#" class="nav-link nav-icon">
+            <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+              <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#search"/>
+            </svg>
+            <span class="visually-hidden">Search</span>
+          </a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link nav-icon position-relative">
+            <span class="position-relative">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+                <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#buy"/>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+              <span class="position-absolute top-0 start-100 translate-middle badge rounded-pill bg-info text-white">
+                1
+                <span class="visually-hidden">shopping basket items</span>
+              </span>
+            </span>
+          </a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link nav-icon">
+            <img src="/docs/5.1/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User" loading="lazy">
+          <span class="visually-hidden">My account</span>
+          </a>
+        </li>
+      </ul>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#global-header" aria-controls="global-header" aria-expanded="false" aria-label="Toggle navigation">
         <span class="navbar-toggler-icon"></span>
       </button>
+    </div>
+    <div class="navbar-collapse collapse" id="global-header" style="height: auto;">
+      <ul class="navbar-nav">
+        <li class="nav-item"><a class="nav-link active" href="#">Discover</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Shop</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Services</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Entertainment</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">News</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Support</a></li>
+      </ul>
+    </div>
+    <ul class="navbar-nav d-none d-md-flex">
```

Comment:
I'm wondering if you shouldn't do something like this here:
```diff
diff --git a/site/content/docs/5.1/examples/navbars/index.html b/site/content/docs/5.1/examples/navbars/index.html
index 689328afd..596bc12a6 100644
--- a/site/content/docs/5.1/examples/navbars/index.html
+++ b/site/content/docs/5.1/examples/navbars/index.html
@@ -70,9 +70,9 @@ extra_css:
         <li class="nav-item"><a class="nav-link" href="#">Support</a></li>
       </ul>
     </div>
-    <ul class="navbar-nav d-none d-md-flex">
+    <ul class="navbar-nav d-none d-md-flex pb-1"><!--TODO: replace .pb-1 with a fixed value more precise-->
       <li class="nav-item">
-        <a href="#" class="nav-link nav-icon pt-1 pb-2">
+        <a href="#" class="nav-link nav-icon">
           <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
             <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#search"/>
           </svg>
@@ -80,13 +80,13 @@ extra_css:
         </a>
       </li>
       <li class="nav-item">
-        <a href="#" class="nav-link nav-icon pt-1 pb-0 position-relative">
+        <a href="#" class="nav-link nav-icon">
+          <svg fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+            <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#buy"/>
+          </svg>
+          <span class="visually-hidden">Basket</span>
           <span class="position-relative">
-            <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
-              <use xlink:href="/docs/5.1/assets/img/boosted-sprite.svg#buy"/>
-            </svg>
-            <span class="visually-hidden">Basket</span>
-            <span class="position-absolute top-0 start-100 translate-middle badge rounded-pill bg-info text-white mt-1">
+            <span class="badge">
               1
               <span class="visually-hidden">shopping basket items</span>
             </span>
@@ -94,7 +94,7 @@ extra_css:
         </a>
       </li>
       <li class="nav-item">
-        <a href="#" class="nav-link nav-icon pt-1 pb-2">
+        <a href="#" class="nav-link nav-icon">
           <img src="/docs/5.1/assets/img/navbar-contact.png" width="25" height="25" role="img" alt="User" loading="lazy">
         <span class="visually-hidden">My account</span>
         </a>
```
Don't push it because I can't right now evaluate the impacts on the minimized version. And it should be modified after in this file and in the doc.